src/image.rs

```diff
@@ -1,15 +1,27 @@
-#[derive(Debug, Clone, RustcEncodable, RustcDecodable)]
+use serde::de::{Deserializer, DeserializeOwned};
+use serde::Deserialize;
+
+fn null_to_default<'de, D, T>(de: D) -> Result<T, D::Error>
+    where D: Deserializer<'de>,
+          T: DeserializeOwned + Default
+{
+    let actual : Option<T> = Option::deserialize(de)?;
+    Ok(actual.unwrap_or_default())
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
 #[allow(non_snake_case)]
 pub struct Image {
     pub Created: u64,
     pub Id: String,
     pub ParentId: String,
+    #[serde(deserialize_with = "null_to_default")]
```

Comment:
Is there a reason why you didn't use `#[serde(default)]` here? I believe that's built in, and this would allow us to get rid us `null_to_default`.

Reply:
Ah, maybe there's a complication with the key `RepoTags` being present, but `null`? I thought that did the right thing. Let's figure this out, and get this merged and released for you.

Reply:
Yes the `null_to_default` is needed because I have seen a response from docker where `RepoTags` is present but `null`. I just now tried to just use `#[serde(default)]` and the test fails for the image response where `RepoTags` is `null`

Reply:
OK, great. Thank you for verifying that!